### PR TITLE
Backend sean merging with the frontend

### PR DESCRIPTION
**Pull request by jcheruvelil**

### CODE REVIEW

**Comment by 5phun** [comment]
Looks good to me

File: packages/express-backend/backend.js [hunk 1]
```diff
@@ -1,19 +1,147 @@
...
+      phoneNumber,
+      cuisines,
+      price,
+      reviews,
+      profilePicture,
+      foodGallery
+    };
+    console.log("cheef", newChef)
+    // Mock function to save chef to the database
+    await chefService.addChef(newChef);
```
Comment (msri-02) [inline packages/express-backend/backend.js:74]: is the mock function supposed to be permanent? was curious how it's different from app.post add chef

File: packages/react-frontend/src/MyApp.jsx [hunk 2]
```diff
@@ -1,19 +1,28 @@
...
 import { useNavigate } from "react-router-dom";
 import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
 import SignUp from "./SignUp";
 import SearchPage from "./SearchPage";
 
 
 function MyApp() {
-  const [chefData, setChefData] = useState([]);
-  const [filteredData, setFilteredData] = useState([]);
+  const [chefData, setChefData] = useState([
```
Comment (msri-02) [inline packages/react-frontend/src/MyApp.jsx:15]: probably need to remove this?

File: packages/react-frontend/src/SearchPage.jsx [hunk 3]
```diff
@@ -3,9 +3,12 @@ import React, { useState, useEffect } from "react";
...
   return (
     <div className="card">
-      <h2>
-        {chef.firstName} {chef.lastName}
-      </h2>
+      <div className="card-header">
+        <h2>
+          {chef.firstName} {chef.lastName}
+        </h2>
+        <img src={chef.profilePicture} alt="No profile picture" className="profile-picture" />
```
Comment (msri-02) [inline packages/react-frontend/src/SearchPage.jsx:10]: just fyi in frontend the className is "chef-image" as it is named in the css file

**Review by msri-02 — COMMENTED** [review]
thanks for taking a look at the cloudinary

File: packages/express-backend/backend.js (hunk 1 above)
Comment (jcheruvelil) [reply packages/express-backend/backend.js:74]: the addChef function is adding it to the mongo database - this addChef function is in the chef-service.js file

**Comment by danielbrathwaite** [comment]
Similar to Sean's PR, we might want to remove some commented out code from PR's, just to keep the origin branches clean. 
I think there's also some unused code, might want to check out if we'll use some of those functions in the future, or if it's safe to remove them. Could follow up in a clean-up PR.

Other than that, LGTM! Nice work!

**Pull request merged**